src/components/pages/Home.tsx

```diff
@@ -24,11 +24,10 @@ async function getQueryHighest(
   chainIds: number[]
 ): Promise<[SearchQuery, string]> {
   const dids = await getHighestLiquidityDIDs(chainIds)
-
   // TODO: this query needs to adapt to chainIds
```

Comment:
I think we did that so can remove the TODO

Reply:
sure gonna remove that